index.php

```diff
@@ -26,6 +26,8 @@
         <button class="mode yellow painter" data-colour="#ffff00" name="yellow"></button>
         <button class="mode orange painter" data-colour="#ffa500" name="orange"></button>
         <button class="mode eraser" name="eraser"></button>
+        <button class="changeBG" name="changeBG"></button>
```

Comment:
I like the way this works, but I think it needs some kind of instructions to explain that you need to click a colour, then the background tool